dcap_quoteverify/types/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+//! Rust wrappers for SGX DCAP quoteverify types
+
+#![no_std]
+
+#[cfg(test)]
+#[macro_use]
+extern crate yare;
+
+use mc_sgx_dcap_quoteverify_sys_types::sgx_qv_path_type_t;
+
+#[derive(Eq, PartialEq, Debug)]
+pub enum Path {
```

Comment:
Since this was a C enum, chose to use a rust enum and not a newtype wrapper

Reply:
Seems reasonable, though one thing I did end up doing in a similar situation was using rust-friendly names, and setting the value: https://github.com/mobilecoinfoundation/mobilecoin/blob/65791210c87d56867545e549b5328aaeba3df331/sgx/core-types/src/error.rs

Reply:
e.g. rather than "QVE" it should be `QuoteVerifierEnclave` and `QuoteProvisioningLibrary`...

Reply:
(oh, and also `#[non_exhaustive]`)